Zuly-Montaño/tarea2/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Revista</title>
+    <link rel="stylesheet" href="index.css">
+</head>
+
+<body>
+    <main class="container">
+        <header>
+            <section class="titulo-flex">
+                <article>
```

Comment:
@zuly7 , please research a little bit about the use of sections and articles

https://medium.com/design-code-repository/html-elements-section-vs-div-vs-article-a8c34e6548cf